src/NzbDrone.Core/DecisionEngine/DownloadDecisionComparer.cs

```diff
@@ -32,6 +32,7 @@ public int Compare(DownloadDecision x, DownloadDecision y)
                 CompareLanguage,
                 ComparePreferredWordScore,
                 CompareProtocol,
+                CompareIndexerPriority,
```

Comment:
I think you need to drop this below CompareEpisodeNumber... CompareEpisodeCount is what ensures that season packs are grabbed first (if all else is equal). Otherwise the user gets into the situation that if not all episodes are available as single downloads, one single episode will prevent the rest from ever being grabbed (since the season pack wouldn't be an upgrade for all episodes).
It'll still prefer a pack from the indexer with a higher priority over a pack from a lower prio. But it won't prefer a single episode from a high prio indexer over a pack from a lower prio.